components/src/core/components/Input/DateInput.vue

```diff
@@ -117,6 +130,10 @@ export default defineComponent({
           : parseDate(this.dateTyped, this.ioformat);
         this.dateTyped = '';
       }
+      const calendarDropdown = (e.target as Element).parentElement?.nextElementSibling;
```

Comment:
why do you have to check like this?

Reply:
The onBlur event on Dateinpu.vue is triggering and closing the Calendar dropdown whenever we are trying to get the focus into Calendar. Since its taking out the focus (blurring) from DateInput. So I ha ve put this check to trigger the Blur Event only whenever the focus is not on one of the calendar dropdowns elements. should I look into any better way we can achieve this?